packages/storybook/stories/va-notification.stories.jsx

```diff
@@ -0,0 +1,220 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
+
+const notificationDocs = getWebComponentDocs('va-notification');
+
+export default {
+  title: 'Components/Notification',
+  id: 'components/va-notification',
+  argTypes: { 
+    headline: {
+      table: {
+        disable: true,
+      },
+    },     
+    children: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+  parameters: {
+    componentSubtitle: `va-notification web component`,
+    docs: {
+      page: () => <StoryDocs data={notificationDocs} />,
+    },
+  },
+};
+
+const defaultArgs = {
+  'visible': true,
+  'symbol': 'none',
+  'close-btn-aria-label': 'Close notification',
+  'closeable': true,
+  'has-border': true,
+  'has-close-text': false,
+  'headline': (
+    <h3 part="headline">Notification heading</h3>
+  ),
+  'children': (
+     <p>Notification body</p>
+  ),
+  'href': 'https://www.va.gov/',
+  'text': 'Manage your notifications',
+};
+
+const Template = ({
+  visible,
+  symbol,
+  href,
+  text,
+  'close-btn-aria-label': closeBtnAriaLabel,
+  closeable,
+  'has-border': hasBorder,
+  'has-close-text': hasCloseText,
+  headline,
+  children,
+}) => {
+  return (
+    <va-notification
+      visible={visible}
+      symbol={symbol}
+      href={href}
+      text={text}
+      closeable={closeable}
+      has-border={hasBorder}
+      has-close-text={hasCloseText}
+      closeBtnAriaLabel={closeBtnAriaLabel}
+    >
+      {headline}
+      {children}
+    </va-notification>
+  )
+};
+
+const MultipleTemplate = ({
+  'close-btn-aria-label': closeBtnAriaLabel,
+  closeable,
+  'has-border': hasBorder,
+}) => {
+  return (
+    <>
+      <va-notification
+        visible="true"
+        symbol="action-required"
+        href="https://www.va.gov/"
+        text="Manage your VA debt"
+        closeable={closeable}
+        has-border={hasBorder}
+        closeBtnAriaLabel={closeBtnAriaLabel}
+        class="vads-u-margin-bottom--1p5"
+      >
+        <h3 part="headline">
+          You have a new education debt.
+        </h3>
+        <p className="vads-u-margin-top--0">
+          <time dateTime="2023-05-1 13:13:00">Wednesday, May 11 at 1:13pm</time>
+        </p>
```

Comment:
Does the notification component always have a dateTime display like this? 

I'm just wondering if this should either be made its own prop or we maybe just make a dedicated named slot for it that we style from within the component if possible (we probably don't want Formation classes used because we're trying to move off of that framework).

Reply:
@allisonlu We had some internal back and forth about this and what we'd like to do is create a named slot for the date/time. I've mocked something up just to make sure I wasn't leading you down the wrong path. But essentially what we'd like to do is manage the styling of this element in the component itself while also allowing some slot flexibility with content itself.

Here's an approach which I think does that but do you mind giving it a try? We essentially want to remove the need for the `p` or any styling classes outside of the component.

![Screenshot 2023-06-01 at 2 46 44 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/febefb93-a267-4102-bb7e-180c9029093c)

![Screenshot 2023-06-01 at 2 46 59 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/e15d07eb-b217-418a-8fb2-f69a6101d8a9)

![Screenshot 2023-06-01 at 2 46 48 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/ef6a471f-0df8-4f68-8c33-f3926ca191d0)